defender/gasless-nft-minting/serverless.yml

```diff
@@ -0,0 +1,41 @@
+service: gasless-nft-minter
+configValidationMode: error
+frameworkVersion: '3'
+
+provider:
+  name: defender
+  stage: ${opt:stage, 'dev'}
+  stackName: gasless_nft_minting
+  ssot: false
+
+custom:
+  config: ${file(./config.${self:provider.stage}.yml)}
+  secrets: ${file(../.secrets/${self:provider.stage}.yml)}
+
+defender:
+  key: ${self:custom.secrets.keys.defender-api-key}
+  secret: ${self:custom.secrets.keys.defender-api-secret}
+
+functions:
```

Comment:
Wouldn't it be better if a relayer is also deployed and hooked into the autotasks? That wat whoever is deploying this only needs to fund that new relayer. They always have the option to switch manually to an already funded relayer

Reply:
It's a chicken and the egg problem. 
- The contract should be deployed by a relayer and the Autotask needs the deployed contract's address during deployment
- If you deploy the contract first, the stack relay won't exist yet
- If you deploy the stack first, then the contracts wort exist yet

This template is based on the Contract Wizard Deployer, which assumes that you might already have a funded relayer that can deploy contracts. For simplicity’s sake, the choice was made to not create an additional relay and instead connect the one that deployed the contracts initially

Reply:
ok, makes sense. Thanks for the detailed explanation!